src/components/sbb-toggle/sbb-toggle.scss

```diff
@@ -0,0 +1,94 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-toggle-width: fit-content;
+  --sbb-toggle-min-width: calc(
+    var(--sbb-toggle-padding-inline) * 2 + var(--sbb-size-icon-ui-small)
+  );
+  --sbb-toggle-selected-option-background-color: var(--sbb-color-white-default);
+  --sbb-toggle-selected-option-border-color: var(--sbb-color-smoke-default);
+  --sbb-toggle-padding-inline: var(--sbb-spacing-responsive-xxxs);
+  --sbb-toggle-animation-duration: var(--sbb-animation-duration-6x);
+  --sbb-toggle-height: #{sbb.px-to-rem-build(28)};
+
+  @include sbb.mq($from: medium) {
+    --sbb-toggle-height: #{sbb.px-to-rem-build(32)};
+  }
+}
+
+:host([even]) {
+  --sbb-toggle-width: 100%;
+
+  ::slotted(sbb-toggle-option) {
+    width: 50%;
+  }
+}
+
+:host([disabled]) {
+  --sbb-toggle-selected-option-background-color: var(--sbb-color-milk-default);
+  --sbb-toggle-selected-option-border-color: var(--sbb-color-graphite-default);
+}
+
+:host([size='m']) {
+  --sbb-toggle-padding-inline: var(--sbb-spacing-responsive-s);
+  --sbb-toggle-height: #{sbb.px-to-rem-build(44)};
+
+  @include sbb.mq($from: medium) {
+    --sbb-toggle-height: #{sbb.px-to-rem-build(52)};
+  }
+}
+
+:host([disable-animation]) {
+  --sbb-toggle-animation-duration: 0s;
+}
+
+.sbb-toggle {
+  @include sbb.text-m--regular;
+
+  position: relative;
+  display: flex;
+  align-items: center;
+  width: var(--sbb-toggle-width);
+  max-width: 100%;
+  min-width: calc(var(--sbb-toggle-min-width) * 2);
+  height: var(--sbb-toggle-height);
+  user-select: none;
+  -webkit-tap-highlight-color: transparent;
+  background: var(--sbb-color-cloud-default);
+  border-radius: var(--sbb-border-radius-16x);
```

Comment:
As the border radius is used at least two times, I'd suggest to extract it into a variable.